vpr/src/route/rr_graph_util.cpp

```diff
@@ -98,6 +98,7 @@ int seg_index_of_sblock(int from_node, int to_node) {
 void reorder_rr_graph_nodes(const t_router_opts& router_opts) {
     auto& device_ctx = g_vpr_ctx.mutable_device();
     auto& graph = device_ctx.rr_nodes;
+    auto& rr_graph = device_ctx.rr_graph;
```

Comment:
After a discussion with @vaughnbetz , we may consider to move this function as an internal method of RRGraphView/RRGraphbuilder.